src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,67 @@
+package qna.domain;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Embeddable
+public class Answers {
+
+    @OneToMany(mappedBy = "question")
+    private List<Answer> answers = new ArrayList<>();
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    protected Answers() {
+
+    }
+
+    public void checkIsOwner(User writer) {
+        for (Answer answer : answers) {
+            checkIsOwner(writer, answer);
+        }
+    }
+
+    private void checkIsOwner(User writer, Answer answer) {
+        if (!answer.isOwner(writer)) {
+            throw new RuntimeException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+        }
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public void delete() {
+        for (Answer answer : answers) {
+            answer.delete();
+        }
+    }
+
+    public List<DeleteHistory> delete(User writer) {
```

Comment:
반환 값으로 일급 컬렉션을 사용해보면 좋을 것 같아요 :)

추가적으로 질문이 지워질때, 답변이 함께 지워져야하므로 삭제 시간은 동일해야하지 않을까요?

```suggestion
    public DeleteHistories delete(User writer, LocalDateTime deletedAt) {
```

Reply:
시간 상 거의 차이가 없을 거라고 생각했는데 말씀하신대로 삭제 시간을 동일하게 둬야 될 것 같네요
꼼꼼한 피드백 감사합니다!